hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.jsx

```diff
@@ -113,13 +135,32 @@ export default function ActionRuleFormColumns({
                 newClassifications[index] = newClassification;
                 onChange({classifications: newClassifications});
               };
+              const onClassificationDelete = () => {
+                const newClassifications = classifications;
+                newClassifications.splice(index, 1);
+                onChange({classifications: newClassifications});
+              };
               return renderClassificationRow(
                 classification,
                 onClassificationChange,
+                onClassificationDelete,
                 index,
               );
             })
           : []}
+        <Button
```

Comment:
This button "dances around", sometimes coming at the end of the label, sometimes not. Maybe put `<br />` just before it, or surround the `<Form.Label>` above at 120 with `<Row><Col>`.